assets/src/dashboard/app/views/myStories/content/storyGridItem/storyDisplayContent.js

```diff
@@ -122,17 +123,21 @@ const StoryDisplayContent = ({
       ) : (
         <Row>
           {isLocked && <LockIcon />}
-          <Title
-            href={titleLink}
-            tabIndex={tabIndex}
-            aria-label={sprintf(
-              /* translators: %s: title*/
-              __('Open %s in editor', 'web-stories'),
-              title
-            )}
-          >
-            {formattedTitle}
-          </Title>
+          {titleLink ? (
+            <TitleLink
+              href={titleLink}
+              tabIndex={tabIndex}
+              aria-label={sprintf(
+                /* translators: %s: title*/
+                __('Open %s in editor', 'web-stories'),
+                title
+              )}
+            >
+              {formattedTitle}
+            </TitleLink>
+          ) : (
+            <Title>{formattedTitle}</Title>
```

Comment:
this needs to be given the proper headline hierarchy (otherwise it defaults to h1):
```suggestion
            <Title as='h3'>{formattedTitle}</Title>
```

Reply:
Should that be done in `components.js` as part of the `attrs()` call?

```js
export const Title = styled(Headline).attrs({
  as: 'h3',
  size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL,
})`
....
`;
```

Reply:
it does the same thing either way.  Since size is getting set in attrs then that's good too.

Reply:
See https://github.com/google/web-stories-wp/pull/8591#discussion_r685414122

Reply:
Regardless of link color discussed in https://github.com/google/web-stories-wp/pull/8591#discussion_r685414122, can you add the `h3` as per the above?

Reply:
Good point. Changed to h3 - https://github.com/google/web-stories-wp/pull/8591/commits/38f6a81caf5ed67b82478e2aa5655cec42680116